core/unit_test/CMakeLists.txt

```diff
@@ -293,5 +294,61 @@ KOKKOS_ADD_EXECUTABLE_AND_TEST(
   SOURCES UnitTestMain.cpp  TestHostBarrier.cpp
 )
 
+FUNCTION (TEST_FEATURE_IMPLEMENTED CURRENT_FEATURE FEATURE_LIST)
+  SET ( FEATURE_IMPLEMENTED OFF PARENT_SCOPE )
+  FOREACH ( IMPLEMENTED_FEATURE ${FEATURE_LIST})
+    message( STATUS "OUT:${IMPLEMENTED_FEATURE}, ${CURRENT_FEATURE}" )
+    if(CURRENT_FEATURE STREQUAL ${IMPLEMENTED_FEATURE} )
+      set (FEATURE_IMPLEMENTED ON PARENT_SCOPE )
+    endif()
+  ENDFOREACH()
+ENDFUNCTION()
+
+
+FUNCTION (KOKKOS_ADD_INCREMENTAL_TEST BACK_END_NAME)
+
+  # Iterate over incremental tests in directory
+    # Call here the bash script to get the file list in the test directory
+    # OUTPUT: List of test file names
+  SET( INCREMENTAL_FILE_LIST ExecSpaceCompilation.hpp )
```

Comment:
Note that this is a place holder...variable INCREMENTAL_FILE_LIST will be populated with script above

Reply:
I don't think we need a bash script to build the file list can't we just do this in cmake?

Reply:
Switched to use GLOB for file list in incremental